data/AppEntryClear.css

```
@@ -0,0 +1,20 @@
+/*
+* SPDX-License-Identifier: GPL-2.0-or-later
+* SPDX-FileCopyrightText: 2017-2024 elementary, Inc. (https://elementary.io)
+*/
+
+@keyframes sweep {
+    0%   { -gtk-icon-transform: rotate(0deg);   }
+    25%  { -gtk-icon-transform: rotate(30deg); }
+    50%  { -gtk-icon-transform: rotate(0deg);   }
+    75%  { -gtk-icon-transform: rotate(-30deg);  }
+    100% { -gtk-icon-transform: rotate(0deg);   }
+}
```

I adjusted the timing a bit and added an x transform so that the motion looks like it's pivoting from the handle more. This way you get kind of a big exaggerated sweep in the direction of the sweep as implied by the icon with a little less rebound at the end

```suggestion
@keyframes sweep {
    0%   { -gtk-icon-transform: rotate(0deg) translatex(0);   }
    20%  { -gtk-icon-transform: rotate(40deg) translatex(-5px); }
    60%  { -gtk-icon-transform: rotate(0deg)   translatex(0); }
    80%  { -gtk-icon-transform: rotate(-30deg) translatex(3px);  }
    100% { -gtk-icon-transform: rotate(0deg) translatex(0);   }
}
```